### PR TITLE
Fix date conversion problem when creating bet

### PR DESCRIPTION
**Pull request by ghost**
Re-upload: accidentally merged with #36 
Pivotal: https://www.pivotaltracker.com/story/show/143166175
Added proper formatting to the jquery-datepicker so it adds user's
timezone to the date string as well.

### CODE REVIEW

File: app/controllers/user_bets_controller.rb [hunk 1]
```diff
@@ -16,4 +18,36 @@ def user_bet_params
     params.require(:user_bet)
           .permit(:user_id, :bet_id, :amount_bet, :bet_option_id)
   end
+
+  def verify_fields
```
Comment (qr8r) [inline app/controllers/user_bets_controller.rb]: The controller is not the correct place to perform validations

File: app/controllers/user_bets_controller.rb [hunk 2]
```diff
@@ -1,4 +1,6 @@
 class UserBetsController < ApplicationController
+  before_action :verify_fields, only: :create
```
Comment (qr8r) [inline app/controllers/user_bets_controller.rb]: Im not sure this commit fits with this PR.

File: app/assets/javascripts/lib/date_picker.js [hunk 3]
```diff
@@ -1,5 +1,6 @@
 $(document).on("turbolinks:load", function(){
   $(".jquery-datepicker").datetimepicker({
-    minDate: "0"
+    minDate: "0",
+    format: "Y-m-d G:i:s P"
```
Comment (qr8r) [inline app/assets/javascripts/lib/date_picker.js:4]: Perhapse for now we might just want to set the rails time zone to be pacific https://www.varvet.com/blog/working-with-time-zones-in-ruby-on-rails/

**Review by qr8r — CHANGES_REQUESTED** [review]

File: app/controllers/user_bets_controller.rb (hunk 2 above)
Comment (ghost) [reply app/controllers/user_bets_controller.rb]: Yeah, was redone in the previous PR, thank you for catching this one.

**Review by qr8r — APPROVED** [review]

**Pull request merged**